examples/use_case/gogtp-k8s/k8s_bridged_networks/README.md

```diff
@@ -0,0 +1,66 @@
+## Deploy GoGTP example with Multus on Kubernetes
+
+This example use-case deploys the GoGTP service chain with Multus support on Kubernetes. The example includes two endpoints ("User Equipment" and "External Server") that can be used to test traffic through the GoGTP EPC implementation.
+
+### Prerequisites
+A Kubernetes cluster must be available prior to running this example. The cluster must be configured with Multus to provide additional interfaces to the pods in the service chain. 
+
+A temporary fix must be applied to the tools/kubernetes_provisioning.sh script for Multus to work:
+```
+vim tools/kubernetes_provisioning.sh
+```
+Update line 54
+```
+-ti crosscloudci/k8s-infra:multus \
```

Comment:
@denverwilliams could we instead change it to be k8s-infra:multus, tag that as a release, then revert the change on master so that the docs and steps are simpler to follow?

Reply:
Fair enough, I'll reset it back to Multus.

Reply:
But I don't want to remove it from the Readme.